docs/index.rst

```diff
@@ -44,18 +53,80 @@ Welcome to NEURON's documentation!
 .. toctree::
    :maxdepth: 1
    :caption: Removed Features
+   :hidden:
 
    removed_features.rst
 
 .. toctree::
    :maxdepth: 1
    :caption: Changelog
+   :hidden:
 
    changelog.md
 
-Indices and tables
-==================
 
-* :ref:`genindex`
-* :ref:`modindex`
-* :ref:`search`
+.. image:: neuron-overview.png
+   :width: 75%
+   :align: center
+   :alt: NEURON image with ShapePlot, code, and cortical column
+
+
+Installation
+------------
+
+.. raw:: html
+
+   <div id="downloadMacOS" style="display:none">
+      <p>
+         On macOS, install via:
+         <code>pip3 install neuron</code>
+      </p>
+      <p>
+         Alternatively, 
+         <a href="https://github.com/neuronsimulator/nrn/releases/download/8.1.0/nrn-8.1.0-macosx-10.9-universal2-py-38-39-310.pkg">download macOS installer</a>
```

Comment:
Would be nice to have a generic link that points to latest pkg (thus avoiding updating the docs after each release)

Reply:
Do we have such a link?

Reply:
did a quick google search and came up with nothing. There was `git.io`  backed by GitHub but it was discontinued.

Reply:
And if I think about, it's not that easy to manage wrt different ReadTheDocs versions..

Reply:
Would this make sense: 
```suggestion
          head over to <a href="https://github.com/neuronsimulator/nrn/releases/">NEURON GitHub Releases</a> and download the desired macOS package installer from `Assets`
```

Reply:
There is `nightly.link` to download artifacts from your workflow runs? Does that help?

Reply:
Not really, because we want official versions to match RTD tags

Reply:
> Would this make sense:

Personally I think install links ought to take you directly to the installer not another page to search through. 

> `nightly.link`

Arguably if you're viewing documentation for version X it should take you to the X installer not the latest.

Reply:
I'm kind of ok with Mac and Linux taking you to a giant page with all versions since you can just pip install them... But for Windows, IMO  we shouldn't make them search for the only way to install it.

Reply:
> Personally I think install links ought to take you directly to the installer not another page to search through.

Totally agree. Just thinking about the extra maintenance here.

Reply:
Agree re: maintenance. It's a real issue; the NEURON website was never updated for 8.0.2 because we just missed doing the update and didn't notice until it was time for 8.1.